crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -89,7 +89,7 @@ impl SequencerService for SequencerServer {
         let rollup_ids = request
             .rollup_ids
             .into_iter()
-            .map(RollupId::try_from_vec)
+            .map(|raw| RollupId::try_from_raw(&raw))
```

Comment:
Similar to my previous comment, using `request.rollup_ids.iter().map(RollupId::try_from_vec)` would make this a tad more elegant.